packages/core/test/helpers/index.ts

```diff
@@ -128,15 +129,12 @@ export function WorkflowExecuteAdditionalData(
 		connections: {},
 	};
 
-	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-	// @ts-ignore
 	return {
-		credentialsHelper: new CredentialsHelper(''),
+		credentialsHelper: new CredentialsHelper(),
 		hooks: new WorkflowHooks(hookFunctions, 'trigger', '1', workflowData),
 		executeWorkflow: async (workflowInfo: IExecuteWorkflowInfo) => {},
-		sendMessageToUI: (message: string) => {},
+		sendDataToUI: (message: string) => {},
```

Comment:
Why rename? The original looks more accurate?

Reply:
this has been renamed in the Interface. This change is only making this code correct, so that `typecheck` passes.